MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -486,11 +486,11 @@ static uint32_t getNextViewMaskGroup(uint32_t viewMask, uint32_t* startView, uin
                                                 bool storeOverride) {
     MTLStoreAction storeAction = getMTLStoreAction(subpass, isRenderingEntireAttachment, hasResolveAttachment, isStencil, storeOverride);
     MVKPixelFormats* pixFmts = _renderPass->getPixelFormats();
-    if (pixFmts->isDepthFormat(pixFmts->getMTLPixelFormat(_info.format)) && !isStencil) {
-        [cmdEncoder->_mtlRenderEncoder setDepthStoreAction: storeAction];
-    } else if (pixFmts->isStencilFormat(pixFmts->getMTLPixelFormat(_info.format)) && isStencil) {
+    if (pixFmts->isStencilFormat(pixFmts->getMTLPixelFormat(_info.format)) && isStencil) {
```

Comment:
Unfortunately, I think this logic is now going to cause `setColorStoreAction:` to be called on a stencil-only format when processing the depth attachment (`isStencil == false`).

Perhaps we should move away from an `else` chain, and implement something more explicit, like:

	MTLPixelFormat mtlFmt = pixFmts->getMTLPixelFormat(_info.format);
	bool isDepthFormat = pixFmts->isDepthFormat(mtlFmt);
	bool isStencilFormat = pixFmts->isStencilFormat(mtlFmt);
	bool isColorFormat = !(isDepthFormat || isStencilFormat);

    if (isColorFormat) {
        [cmdEncoder->_mtlRenderEncoder setColorStoreAction: storeAction atIndex: caIdx];
    }
    if (isDepthFormat && !isStencil) {
        [cmdEncoder->_mtlRenderEncoder setDepthStoreAction: storeAction];
    }
    if (isStencilFormat && isStencil) {
        [cmdEncoder->_mtlRenderEncoder setStencilStoreAction: storeAction];
    }

Reply:
I added the check for color and nice named intermediates, but kept the `else`'s as they make it clearer that at most one case will occur.
One little usability thing is maybe it should do nothing for color formats if isStencil is set true?